moto/events/models.py

```diff
@@ -55,6 +55,24 @@ def remove_targets(self, ids):
             if index is not None:
                 self.targets.pop(index)
 
+    @classmethod
+    def create_from_cloudformation_json(
+        cls, resource_name, cloudformation_json, region_name
+    ):
+        properties = cloudformation_json["Properties"]
+        event_backend = events_backends[region_name]
+        event_name = properties.get("Name") or resource_name
```

Comment:
Change this to ```properties.get("Name", resource_name)``` please - same in the delete. Slightly more readable IMO

Reply:
I used this "safer" approach to handle the case where `{"Name": ""}`. Is it possible ?

Reply:
That is a very valid point! Didn't think about that possibility. Leave it as is then